index.html

```diff
@@ -192,6 +192,22 @@ <h5 class="m-l">HTML</h5>
         </div>
       </div>
     </section>
+    <footer>
+      <div class="mycontainer">
+        <h2>Contact me</h2>
+        <p>If you have an application you are interested in developing, a feature that you
+          need built or a project that you needs coding. I'd love to help with it
+        </p>
+      </div>
+      <div class="form-taget">
+      <form action="#" class="form" method="post">
```

Comment:
- [ ] please ensure that you are to Collect data by using the [Formspree service](https://formspree.io/html). therefore your Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
action value should be similar to https://formspree.io/f/{form_id}
method value equals POST like you have added.